packages/atomic/cypress/integration/search-box.cypress.ts

```diff
@@ -6,73 +6,191 @@ import {
 } from '../selectors/search-box-selectors';
 
 const queryText = 'test';
-const htmlCode = '<atomic-search-box></atomic-search-box>';
 
 describe('Search Box Test Suites', () => {
-  beforeEach(() => {
-    setUpPage(htmlCode);
-    cy.wait(500);
-    generateAliasForSearchBox();
-  });
-
-  it('should load', () => {
+  function testLoad() {
```

Comment:
Just as a general question, how fun/nice/easy is it to write and add tests with Cypress ?

Reply:
I thought it was pretty good, since it's working on the rendered DOM there is a little bit of inconsistency and unpredictability in terms of what its actually going to do. For example, while writing these tests I noticed if I aliased the search button, it would click the clear button instead for some reason but if I just called `.find('submit-button')` every time it worked. But overall I liked it, writing tests once you understand the quirks of cypress is pretty quick.